### ISSUE TITLE
Java Talk by Andres Almiray

### ISSUE THREAD

**Issue by carwyn** [NorthWalesTech/NorthWalesTech#65]
From @jonbullock in #42 

> I put Andres Almiray in contact with the Manchester Java Community and he's coming over to do a talk in August: https://www.meetup.com/ManchesterUK-Java-Community/events/238223376/
> 
> He indicated he was going to stay for a few days... is it worth asking if he'd come and do a talk at NWT as well?

Proposed Date: Aug 30th or Sept 1st 7pm to 9pm
Location: TBD
Considerations, Byw a Bod Project Conference is on Sept 1st in Pontio PL2, not sure what time yet.

**Comment by carwyn** [comment]
@tamslinn likes the idea of "Java libraries you can't afford to miss". @davehun and @robshep likely to be interested in this.

Won't let me assign this to you @jonbullock

**Comment by jonbullock** [comment]
I'm sure GitHub has changed their security system recently, I had to grant users write access to the repo to enable this.

**Comment by robshep** [comment]
hooray for "Java libraries you can't afford to miss", not 'cos you can't assign to JB! :)

**Comment by carwyn** [comment]
Aha, adding the contributors team as read only to the repo did it (of course!?). Will make it write access, was just curious :)

**Comment by jonbullock** [comment]
Just heard back from Andres, he's not travelling over to Manchester now, he's doing a remote session instead. 

Oh well, maybe next time.

**Comment by tamslinn** [comment]
Oh that's a shame I was looking forward to that! Will you be going to the remote session? Would be interesting to see how it works and if that would work for NWT sometime too.

**Comment by jonbullock** [comment]
Yeah I'll probably head over for the session. I was going to see how it works for them first as I'm sure he'd be up for a remote session for NWT as he's pushing these on Twitter at the moment.

**Comment by tamslinn** [comment]
Fab that sounds sensible. If remote sessions work well that opens doors. 
Do you know which topic they picked by the way? I seem to recall there was a poll.

**Comment by jonbullock** [comment]
Members are still voting: https://www.meetup.com/ManchesterUK-Java-Community/polls/1249674/ but the Java libs one is in the lead still.

Oh, I've just noticed the NWT MicroBit event is on the same night!

**Comment by tamslinn** [comment]
Oh bother hadn't noticed that :(

Java libs would be my choice too

**Comment by carwyn** [comment]
Oops, we can play the stream in CoWork at the same time as the Micro:bit workshop, the area is big enough.

**Comment by tamslinn** [comment]
I think see how a remote session works for MJC and if it's good try one of our own? Otherwise I'll be torn!

**Comment by carwyn** [comment]
We've been using remote Hangouts for the Fusion 360 workshops, as long as the speaker has a stable connection they work really well.

**Comment by jonbullock** [comment]
Yeah I was thinking the same. We could run a survey for the talk as well if we wanted. Might encourage more participation if you get a say in the talk given.

I'll go along and see how it goes, I can't not go now as I was the instigator :) Shame though as I would have liked an excuse to have a play with my micro:bit, anyway have fun!

**Comment by carwyn** [comment]
Hoping to make the BYOD events a regular thing .. there will be more Micro:bits :) pick a Thursday in September @jonbullock and we'll arrange it specially for you ;)

**Comment by jonbullock** [comment]
I'll get in touch with Andres again to ask about doing a remote session this year... any suggestions on potential months I can put to him?
